app/controllers/admin/submissions_controller.rb

```diff
@@ -66,7 +66,9 @@ def create
       @submission =
         SubmissionService.create_submission(
           submission_params.merge(state: 'submitted'),
-          submission_params[:user_id]
+          submission_params[:user_id],
+          is_admin: AdminUser.exists?,
```

Comment:
I don't think it needs to be passed as a parameter
you are already passing current_user and by it you can determine whether the user is an admin
`AdminUser.exists?(gravity_user_id: current_user)`